<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewPzSetErrorForm</class>
 <widget class="QDialog" name="ViewPzSetErrorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>错误级别：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbLevel"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnInspect">
       <property name="text">
        <string>检查</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tw">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>级别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>凭证号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>分录号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>错误类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>错误说明</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>索引号</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblState">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
